meson.build

```diff
@@ -191,11 +191,11 @@ if not get_option('snpe-support').disabled()
       message('Not Supported System & Architecture. Linux x86_64 is required for snpe sub-plugin')
     else
       # Check whether $SNPE_ROOT (where SNPE SDK is located) is set.
-      cmd = run_command('sh', '-c', 'echo $SNPE_ROOT')
+      cmd = run_command('sh', '-c', 'echo $SNPE_ROOT', check : true)
       SNPE_ROOT = cmd.stdout().strip()
       if SNPE_ROOT != ''
         message('Got $SNPE_ROOT: @0@'.format(SNPE_ROOT))
-        snpe_dir_not_exist = run_command('[', '-d', SNPE_ROOT, ']').returncode()
+        snpe_dir_not_exist = run_command('[', '-d', SNPE_ROOT, ']', check : true).returncode()
         if snpe_dir_not_exist != 0
           error('@0@ does not exists'.format(SNPE_ROOT))
```

Comment:
This error message can be removed again, if you use `check: true`.

However again, you can use the fs module's `fs_mod.is_dir(SNPE_ROOT)` instead.

Reply:
The FS (filesystem) module requires bumping up to 0.53.0 so that I left a TODO comment as you commented.